src/components/container/ContainerComponent.tsx

```diff
@@ -215,7 +215,7 @@ export function ContainerComponent<TData = unknown>(
           }
         }}
         onPointerDown={(event) => {
-          event.preventDefault();
+          // event.preventDefault();
```

Comment:
I prefer to remove code. Or if you keep it commented, add another line comment to explain.

Reply:
I went a bit fast